src/test/java/com/nerdtranslator/translateapibridge/service/NerdTranslatorServiceTest.java

```diff
@@ -0,0 +1,199 @@
+package com.nerdtranslator.translateapibridge.service;
+
+import com.nerdtranslator.translateapibridge.data.TranslationData;
+import com.nerdtranslator.translateapibridge.util.PartOfSpeech;
+import lombok.RequiredArgsConstructor;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+@TestConstructor(autowireMode = TestConstructor.AutowireMode.ALL)
+@RequiredArgsConstructor
+class NerdTranslatorServiceTest {
+
+    private final NerdTranslatorService nerdTranslatorService;
+
+    @Test
+    void oneEnglishWordShouldEqualToAnotherRussianWordCase() {
```

Comment:
@Electron3D How do you think if this and all other test cases names are in correct format? 
For me they look clear but it's slightly different from the existing naming conventions, for example, here: https://dzone.com/articles/7-popular-unit-test-naming

Reply:
first of all it's integration tests, we've had a conversation with Eugeniy about test naming yesterday, my mentors on courses were teaching us to name integration tests kinda like that: "should_return_***_IT" or if we have unit tests something like "methodWeTest_ProvidedParams_ExpectedReturn"